style/ruby/README.md

```diff
@@ -26,6 +26,7 @@ Ruby
   calls.
 * Prefer `if` over `unless`.
 * Use `_` for unused block parameters.
+* Prifix unused variables, or method arguments with `_`.
```

Comment:
No comma.

Instead of "method arguments", how about "parameters"?

And "Prefix" is misspelled.

> Prefix unused variables or parameters with underscore (`_`).